### PR TITLE
Zeal UI component library

### PR DESCRIPTION
**Pull request by mvganeshkumar06**

### CODE REVIEW

File: src/components/alert/Alert.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+    if (type === "warning") {
+        return colors.orange[2];
+    }
+    if (type === "danger") {
+        return colors.red[2];
+    }
+    if (type === "info") {
+        return colors.blue[2];
+    }
```
Comment (jay-rajput30) [inline src/components/alert/Alert.js:17]: could have used switch case here, would have got a default style to set as well.

**Review by jay-rajput30 — COMMENTED** [review]
very well written code.  I like that you have used styled components.  I felt at few places where switch case might have looked better than multiple if statements but again just a minor thing.  Overall awesome work!

**Review by singhadi672 — COMMENTED** [review]
Hi @mvganeshkumar06  the new changes looks awesome!......I surely learned a lot by seeing your code!....and the product also is looking good!

**Comment by jay-rajput30** [comment]
overall the written is very good.  you can merge PR.  keep improving the library.

**Comment by mvganeshkumar06** [comment]
Sure, I'm closing the PR.

**Pull request merged**